src/virtual_renderer.js

```diff
@@ -1620,6 +1620,24 @@ class VirtualRenderer {
                 className: "ace_ghost_text"
             };
             this.session.widgetManager.addLineWidget(this.$ghostTextWidget);
+
+            // Check wether the line widget fits in the part of the screen currently in view
+            var pixelPosition = this.$cursorLayer.getPixelPosition(insertPosition, true);
+            var el = this.container;
+            var height = el.clientHeight;
```

Comment:
`clientHeight` might be rounded down. Maybe using `getBoundingClientRect` would be a little more robust. Though I'm not sure how prominent fractional heights are in ace, if at all.

From https://developer.mozilla.org/en-US/docs/Web/API/Element/clientHeight

Reply:
changed this